AlfiyaSiddique/TastyTrails
# Enhance Card Image Responsiveness on Small Screens (#327)

What does this PR do?
This PR improves the responsiveness of the card image on small screens to maintain a balanced layout, fixing layout issues on mobile devices where the image covers excessive space.
Fixes #327.

How should this be tested?
Test 1: Resize browser window below 750px width and verify the card image resizes proportionately to maintain a balanced layout.
Test 2: Check that the image and other card elements adjust without causing overflow or unbalanced space on smaller screens.

Checklist
